enlighten/scope/LaserControlFeature.py

```diff
@@ -237,6 +242,18 @@ def toggle_laser(self):
         log.debug("toggle_laser called")
         self.toggle_callback()
 
+    def cant_fire_because_battery(self, spec=None):
+        if spec is None:
+            spec = self.ctl.multispec.current_spectrometer()
+        if spec is None:
+            return False
+        
+        perc = self.ctl.battery_feature.get_perc(spec)
+        if perc is None:
+            return False
+
+        return perc < self.MIN_BATTERY_PERC
```

Comment:
Encapsulated this within LaserControlFeature rather than leaving scattered bits around other places like StatusIndicators